pyfr/solvers/euler/kernels/entropyfilter.mako

```diff
@@ -0,0 +1,184 @@
+# -*- coding: utf-8 -*-
+<%inherit file='base'/>
+<%namespace module='pyfr.backends.base.makoutil' name='pyfr'/>
+<%include file='pyfr.solvers.euler.kernels.entropy'/>
+
+<% inf = 1e20 %>
+<% ill_tol = 1e-6 %>
+<% zeta_tol = 1e-3 %>
+<% niters = 20 %>
```

Comment:
Should this be made configurable?

Reply:
Maybe zeta_tol? I'm for leaving as few knobs to tweak as possible though

Reply:
I'd have them read from the config file but not documented.  Makes it easy to change for those who know what they are doing, but avoids people with no clue changing things for the sake of it.